nifi-nar-bundles/nifi-druid-bundle/nifi-druid-bundle-nar/pom.xml

```diff
@@ -0,0 +1,39 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements. See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License. You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+  	<groupId>org.apache.nifi</groupId>
+  	<artifactId>nifi-druid-bundle</artifactId>
+  	<version>1.0-SNAPSHOT</version>
+  </parent>
+  
+  <artifactId>nifi-druid-bundle-nar</artifactId>
+  <packaging>nar</packaging>
+  
+	<dependencies>
+   		<dependency>
+			<groupId>org.apache.nifi</groupId>
+			<artifactId>nifi-druid-controller-service</artifactId>
+			<version>1.0-SNAPSHOT</version>
```

Comment:
Please ensure the version matches the current version we'd merge it into which is 1.5.0-SNAPSHOT
...same for many places of course.